front/cypress/e2e/sign_in.cy.ts

```diff
@@ -19,30 +21,31 @@ describe('Sign in page', () => {
     cy.get('#password').type('test');
     cy.get('#e2e-signin-password-submit-button').click();
 
-    const emailErrorMessage = cy.get('#email').siblings('.e2e-input-error');
-    expect(emailErrorMessage).to.exist;
+    cy.get('#e2e-email-container').should('exist');
+    cy.get('#e2e-email-container').within(() => {
+      cy.get('[data-testid="error-message-text"]').should('exist');
+    });
   });
 
   it('shows an error when no valid email is provided', () => {
-    cy.get('#email').type('test');
+    cy.get('#email').type('test@t');
```

Comment:
Was this test failing when it was just 'test'? If so then maybe the validation is incorrect

Reply:
No this one was failing and skipped, but I re-enabled it a few weeks ago :) When just "test" is used, it's not actually testing our validation. It actually shows the generic error from the email input itself. But the test was trying to check if our validation message was showing (not the generic one from the input). 
Email default error which comes from the input itself: ![image](https://user-images.githubusercontent.com/33987955/234824435-e9db73db-e7d3-4ac1-8114-904af3d3a32c.png)

So I changed it to "test@t" which triggers _our_ validation - which I think was the original intention behind the test :thinking: